### ISSUE TITLE
"A game [for]" platforms string

### ISSUE THREAD

**Issue by aaronfranke** [itchio/itch#2168]
![game](https://user-images.githubusercontent.com/1646875/46913532-c989eb00-cf5c-11e8-8295-bd3db67d22a3.png)

Currently it reads like "A game Windows, Mac, Linux". But it should be "A game for Windows, Mac, Linux". The word "for" should be added. I found the string [here](https://github.com/itchio/itch/blob/master/src/static/locales/en.json) but I'm not submitting a PR because I don't know if this string is used elsewhere and it should be changed for other languages too. Plus there's other strings that should be changed "A utility [for]". Really anything that has the list of platforms after it should have "for".

Yes I'm a nitpicker.

**Comment by fasterthanlime** [comment]
I created a new issue tag just for this 💖

I fully expect that string to disappear and be replaced by more interesting info such as:

  * The game's features (cloud saves, achievements, leaderboards)
  * The game's tags
  * The game's overall rating
  * Whether the game is in any collection you're the creator of / subscribed to
  * Something else along those lines

so I might let that sit for a while.

---

If I were to actually address it now, I'd point out that it is indeed used in two locations, and one of them is tricky: https://github.com/itchio/itch/blob/657399674893d068f6937907357c9a2f016fdcab/src/renderer/basics/GameStats.tsx#L83-L99

`showPlatforms` could be false, so we'd just end up with "A game for", which is no good.

Having another string just for "for" is not friendly for all languages either (and would trigger endless questions on weblate, I'm sure).

---

TL;DR i18n is hard, this is probably a gonner anyway, I'll let you know if you win the award.
